examples/03_connectivity/plot_sphere_based_connectome.py

```diff
@@ -15,6 +16,11 @@
 [2] Dosenbach N.U., Nardos B., et al. "Prediction of individual brain maturity
 using fMRI.", 2010, Science 329, 1358-1361.
 
+[3] `Seitzman, B. A., et al. "A set of functionally-defined brain regions with
+improved representation of the subcortex and cerebellum.", 2018, bioRxiv,
+450452
+<http://doi.org/10.1101/450452>`_
+
 """
 
 ###############################################################################
```

Comment:
Hi @KamalakerDadi @kchawla-pi 
Do you have an idea why the section heading 'Load fMRI data and Power atlas' is not rendering correctly? The html looks like this:
```
[...]
bioRxiv, 450452 Load fMRI data and Power atlas ——————————
We [...]
```

Thanks

Reply:
Giving me a snapshot would help ?

Could you give me a snapshot from your rendered documentation ?

Reply:
I build the doc on my laptop. It looks ok to me.

![pr](https://user-images.githubusercontent.com/11410385/56352228-103bf480-61cf-11e9-845b-77658fc324a7.png)

Reply:
Thanks. Might be an issue with my installation then.
![Bild 18 04 19 um 11 42](https://user-images.githubusercontent.com/5772811/56352283-2ea1f000-61cf-11e9-848f-214935300984.jpeg)

Reply:
The other headings render as expected, btw. It's just this one making problems.